src/room.js

```diff
@@ -78,7 +82,7 @@ export default (onPeer, onSelfLeave) => {
     const typePadded = decodeBytes(typeBytes)
 
     if (actions[typePadded]) {
-      throw mkErr(`action '${type}' already registered`)
+      return actions[typePadded]
```

Comment:
Would it make sense to call `getAction` here so the logic is consolidated?

```suggestion
      return getAction(typePadded)
```

Reply:
Great call! Thank you @jeremyckahn.